src/mixins/amalgamation-mixin.ts

```diff
@@ -41,6 +45,87 @@ export default class AmalgamationMixin extends Vue {
     )
   }
 
+  /**
+   * Get the business information, mailing address, email, and first filing if in LEAR.
+   * Otherwise, return error.
+   */
+  async fetchBusinessInfoForTing (item: any): Promise<any> {
```

Comment:
Didn't know what to name this. I wanted to name it something to indicate that we're fetching information needed for a TING business. This is what I came up with. I'm open for other suggestions though.

Reply:
I don't mind it, except it's fetching more than business info.

Maybe have it similar to `refetchAmalgamatingBusinessesInfo()` below?

Reply:
Or similar to `fetchTingInfo()`?

Reply:
OK, ok, I don't have a strong opinion here 😆

Reply:
I changed it to `fetchAmalgamatingBusinessInfo()`, to have similar naming. Thoughts Sev?

Reply:
I have no more thoughts today.

(OK, it's not bad.)